docs/book/start/logging-packets.md

```diff
@@ -8,88 +8,18 @@ Let's expand this program to log the traffic that is being permitted in the user
 
     Full code for the example in this chapter is available [here](https://github.com/aya-rs/book/tree/main/examples/xdp-log)
 
-## Getting Data to User-Space
+## Logging Data
 
-### Sharing Data
-
-To get data from kernel-space to user-space we use an eBPF map. There are numerous types of maps to chose from, but in this example we'll be using a PerfEventArray.
+To log the data in eBPF program, we are going to use aya-log library, which sends the log message and
```

Comment:
We were already using aya-log. This whole "Sharing Data" section should probably go